### DISCUSSION TITLE
Soul Shards stack of 5

### DISCUSSION THREAD

**Discussion by detrol2k** [chromiecraft/chromiecraft#7237] [Suggestions]
Please end our suffering and make Shards stack to at least 5. :D

**Comment by Helias** [comment]
Hi, we discussed making it stackable or not, but we think that we could change the game's design a bit that has not been changed in several wow expansions despite several discussions and requests that you can also find in the retail forums, on Reddit, and so on. We strongly encourage you to get the soul bags ( https://wowgaming.altervista.org/aowow/?item=6265#can-be-placed-in ) to improve your inventory and keep in mind that you can easily get the soul shard by honor points using the NPC Nyeriah, you can find it in:
- Shattrath City next to the flight master
- Stormwind next to the PvP vendors, in Old Town
- Orgrimmar next to the PvP vendors, in Valley of Strength
- Booty Bay

![image](https://github.com/user-attachments/assets/81c34e12-a0bc-4484-a046-dd22fd45a73a)
